jobs/build-charms/charmcraft-lib.sh

```diff
@@ -13,7 +13,7 @@ ci_charmcraft_launch()
   # Launch local LXD container to publish to charmcraft
   local charmcraft_lxc=$1
   ci_lxc_launch ubuntu:20.04 $charmcraft_lxc
-  until sudo lxc shell $charmcraft_lxc -- bash -c 'snap install charmcraft --classic'; do
+  until sudo lxc shell $charmcraft_lxc -- bash -c "snap install charmcraft --classic --channel=2.x/stable"; do
```

Comment:
on the jenkins runner, start a container for all the charm builds in a particular run and force them to use 2.x/stable.